### PR TITLE
add bandit pip install

### PR DESCRIPTION
**Pull request by sometheycallme**
before this commit bandit does not exist in travis.yml

after this commit bandit is setup as pre-requisite test step
bandit must be installed in order to run python code scans

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,11 @@ before_script:
...
+  - pip install bandit
 
 script:
   - python tests/test.py
   - coverage run tests/test.py
 
 after_success:
-  coveralls
\ No newline at end of file
+  coveralls
```
Comment (kalpatel01) [inline .travis.yml]: Is there meant to be a '-' with this?

Comment (sometheycallme) [inline .travis.yml]: yes.  must have gotten snipped by the cr.

Comment (sometheycallme) [inline .travis.yml]: actually - after looking at master it appears no.
https://github.com/Bridgewater/appetite/blob/master/.travis.yml

Comment (kalpatel01) [inline .travis.yml]: Weird, currently coveralls isn't syncing (https://www.andreagrandi.it/2014/03/02/travis-ci-org-and-coveralls-io-qa-made-easy/), its missing.  Add that and I'll push.

Comment (sometheycallme) [inline .travis.yml]: gotcha so perhaps it was missing.

**Comment by sometheycallme** [comment]
@kalpatel01 - had some merge conflicts to work through.  I'm going to abandon this PR and make a clean commit to get better history.

**Pull request closed without merging**